frontend/src/components/pages/UserCodeReviewPerformance/utils.js

```diff
@@ -2,3 +2,9 @@ export function formatTimeString(timestamp) {
   const date = new Date(Date.parse(timestamp));
   return new Intl.DateTimeFormat().format(date);
 }
+
+export function getReviewStatus({ review }) {
+  return review.state === "changes_requested"
+    ? review.state.split("_").join(" ")
+    : review.state;
+}
```

Comment:
I agree hardcoding can be avoided here. I aslo think that this could be a generic `replaceUnderScoresWithSpaces(str)` function because that what it essentially does 

- there doesn't need to be a check to see if a status is what way.
- there's also no need to convert it to an array. you can use `replaceAll` string method